enabled: True
python: |
  def while_try_break():
    res = []
    arr = [1, 2, 4, 0, 8]
    i = 0
    
    while True:
      try:
        result = 16 // arr[i]
        res.append(result)
        i += 1
      except ZeroDivisionError:
        print("Sorry ! You are dividing by zero ")
        break
    
    return res == [16, 8, 4]